.github/workflows/main.yml

```diff
@@ -0,0 +1,29 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
I think this only runs when the pull request head branch is `master`, which is why it isn't working for this pull request.

Reply:
works on my fork pr https://github.com/stephengroat/MonitorControl/pull/1, maybe it has to be merged in?

Reply:
Yes I think that this should work, CI should run when changes are merged into master and for each PR. It's probably indeed due to needing to be merged in first.